webclient/core/views.py

```diff
@@ -795,7 +795,6 @@ def check_filter_permission(self, qs, action=None):
                 qs = new_qs | qs.filter(**filterdoc)
             else:
                 qs = new_qs
-        qs = qs.cache()
```

Comment:
@pesikj - toto si pridaval ty, robi to problemy ze sa nerefreshuje qs a tiez sposobuje #1810

Reply:
Ok tak vyzkoušíme výkon těch tabulek bez cache.